lib/OAuth2Client.js

```diff
@@ -300,6 +296,10 @@ class OAuth2Client extends EventEmitter {
       opts,
     } = await this.onBuildRequest(req);
 
+    if(this._refreshingToken){
```

Comment:
Seems like invalid formatting to me.

Reply:
Yes, apparently the eslint config was silently invalid